Rubberduck.RegexAssistant/Atom.cs

```diff
@@ -0,0 +1,284 @@
+                    {
+                        return string.Format(AssistantResources.AtomDescription_Literal_EscapedLiteral, relevant);
+                    }
+                    else if (char.IsDigit(relevant[0]))
+                    {
+                        return string.Format(AssistantResources.AtomDescription_Literal_Backreference, relevant);
+                    }
+                    else
+                    {
+                        // special escapes here
+                        switch (relevant[0])
+                        {
+                            case 'd':
+                                return AssistantResources.AtomDescription_Digit;
+                            case 'D':
+                                return AssistantResources.AtomDescription_NonDigit;
+                            case 'b':
+                                return AssistantResources.AtomDescription_WordBoundary;
+                            case 'B':
+                                return AssistantResources.AtomDescription_NonWordBoundary;
+                            case 'w':
+                                return AssistantResources.AtomDescription_WordCharacter;
+                            case 'W':
+                                return AssistantResources.AtomDescription_NonWordCharacter;
+                            case 's':
+                                return AssistantResources.AtomDescription_Whitespace;
+                            case 'S':
+                                return AssistantResources.AtomDescription_NonWhitespace;
+                            case 'n':
+                                return AssistantResources.AtomDescription_Newline;
+                            case 'r':
+                                return AssistantResources.AtomDescription_CarriageReturn;
+                            case 'f':
+                                return AssistantResources.AtomDescription_FormFeed;
+                            case 'v':
+                                return AssistantResources.AtomDescription_VTab;
+                            case 't':
+                                return AssistantResources.AtomDescription_HTab;
+                            default:
+                                // shouldn't ever happen, so we blow it all up
+                                throw new InvalidOperationException("took an escape sequence that shouldn't exist");
+                        }
+                    }
+                }
+                else
+                {
+                    if (specifier.Equals("."))
+                    {
+                        return AssistantResources.AtomDescription_Dot;
+                    }
+                    // Behaviour with "." needs fix
+                    return string.Format(AssistantResources.AtomDescription_Literal_ActualLiteral, specifier);
+                }
+
+                throw new NotImplementedException();
```

Comment:
remnant. theoretically this code should never be reached. I'll fix that, good catch